src/main.rs

```diff
@@ -131,6 +134,53 @@ pub enum CreateCmd {
     },
 }
 
+#[derive(Debug, StructOpt)]
+/// Create or Redeem from an HTLC address
+pub enum HtlcCmd {
+    /// Creates a new HTLC address with a specified hashlock and timelock (in block height), and transfers a value of tokens to it
+    Create {
```

Comment:
Could you do the same thing that master does for the `pay` command? I.e. have a `--commit` option that does the actual create or redeem and without it it just prints what it would end up creating (without the hash and nonce)?